fasm2bels/models/models_data/gtp_common_data.py

```diff
@@ -0,0 +1,133 @@
+params = {
```

Comment:
Does this input come from prjxray directly?  If so, please add this to prjxray-db instead of copying it here.

Reply:
Sure, should be doable. The input is slightly different, as not all the ports were added in the xray one. I'd need to update first xray to push the parameters and ports definitions in the db.

Reply:
This is done